_docs/bigquery/creating-bigquery-dataset.md

```diff
@@ -3,10 +3,26 @@ layout: page
 section: bigquery
 navigation_title: "Creating BigQuery dataset"
 title: "Creating BigQuery dataset"
-order: 4
-date: 2020-04-22
+order: 5
+date: 2020-04-29
 ---
 
 <!---
 In this article explained how to create Google BigQuery dataset
--->
\ No newline at end of file
+-->
+To store data in BigQuery you need to create a [dataset](https://cloud.google.com/bigquery/docs/datasets-intro){:target="_blank"}:
+
+1. Go to the [Google Cloud Platform Console](https://console.cloud.google.com/){:target="_blank"}.
+2. Open the console left side menu and click BigQuery.
+![](/img/bigquery_dataset.1.png)
+3. Select the project where you would like to create a dataset in the bottom-left menu and click CREATE DATASET.
```

Comment:
CREATE_DATASET should be bold. Check the original article for all formatting. General rule - all the labels should be in bold